blazor/diagram/connectors/segments/bezier/bezier.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title: Bezier Segment in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about Bezier Segments in Syncfusion Blazor Diagram component and much more details.
```

Comment:
description: Check out and learn about Bezier Segments in Syncfusion Blazor Diagram component and much more details.